axelor-account/src/main/java/com/axelor/apps/account/web/FixedAssetController.java

```diff
@@ -440,12 +443,20 @@ public void initSplitWizardValues(ActionRequest request, ActionResponse response
               (String)
                   ((LinkedHashMap<String, Object>) request.getContext().get("_fixedAsset"))
                       .get("qty"));
+      Company company =
+          Beans.get(FixedAssetRepository.class)
+              .find(Long.parseLong(request.getContext().get("_id").toString()))
+              .getCompany();
 
       response.setAttr(
           "splitTypeSelect",
           "value",
           qty.compareTo(BigDecimal.ONE) == 0 ? FixedAssetRepository.SPLIT_TYPE_AMOUNT : 0);
       response.setAttr("splitTypeSelect", "readonly", qty.compareTo(BigDecimal.ONE) == 0);
+      response.setAttr(
+          "grossValue",
+          "scale",
+          Beans.get(CurrencyScaleServiceAccount.class).getCompanyScale(company));
```

Comment:
Maybe this should be in `FixedAssetAttrsService`

Reply:
Resolved